components/checkbox.json

```diff
@@ -0,0 +1,48 @@
+{
+  "checkbox": {
+    "checked":{
+		"#normal":{
+			"background": "@theme-control-active-normal",
+			"text": "@theme-text-primary-normal",
+			"border": "@theme-control-active-normal"
```

Comment:
Can you do this as "border": "none" please?

Reply:
Yes. Updated. 👍